### ISSUE TITLE
parallel图坐标值过长

### ISSUE THREAD

**Issue by lvybriage** [apache/incubator-echarts#9749]
option = {
    parallelAxis: [
        {dim: 0, name: 'Price'},
        {dim: 1, name: 'Net Weight'},
        {
            dim: 3,
            name: 'Score',
            type: 'category',
            data: ["tf.nn.relu(conv2d(x_image, w_conv1) + b_conv1)", 'Good', 'OK', 'Bad']
        },
        {dim: 2, name: 'Amount'},
    ],
    series: {
        type: 'parallel',
        parallel: {
                parallelAxisDefault: {
                    axisLabel: {
                        rich: {
                            width: 30
                        }
                    }
                }
            },
        lineStyle: {
            width: 4
        },
        data: [
            [12.99, 100, 82, 'Good'],
            [9.99, 80, 77, 'OK'],
            [20, 120, 60, 'tf.nn.relu(conv2d(x_image, w_conv1) + b_conv1)']
        ]
    }
};
![image](https://user-images.githubusercontent.com/35484733/51098609-7bf47700-1806-11e9-8965-6c327fc8810b.png)

期待的效果： tf.nn.relu(conv2d(x_image, w_conv1) + b_conv1)可以换行显示或者有省略号，hover时提示全文。

**Comment by deqingli** [comment]
@lvybriage  You can format the `parallelAxis` label.  https://www.echartsjs.com/option.html#parallelAxis.axisLabel.formatter
